From d6e9232a751d5ea04019cc09a2bbe0e1e3d04f07 Mon Sep 17 00:00:00 2001
From: timor <timor.dd@googlemail.com>
Date: Sun, 28 Mar 2021 18:18:45 +0200
Subject: [PATCH 12/12] Remove update-checking and recompilation magic

This tries to delete compiled files which are in the store.
---
 core/core-spacemacs.el |  5 ++---
 init.el                | 12 ------------
 2 files changed, 2 insertions(+), 15 deletions(-)

diff --git a/core/core-spacemacs.el b/core/core-spacemacs.el
index afde1b364..24fb285ce 100644
--- a/core/core-spacemacs.el
+++ b/core/core-spacemacs.el
@@ -291,9 +291,8 @@ Note: the hooked function is not executed when in dumped mode."
        (setq read-process-output-max dotspacemacs-read-process-output-max))))
 
   (let ((default-directory spacemacs-start-directory))
-    (if dotspacemacs-byte-compile
-        (spacemacs//ensure-byte-compilation spacemacs--compiled-files)
-      (spacemacs//remove-byte-compiled-files-in-dir spacemacs-core-directory)))
+    (when dotspacemacs-byte-compile
+        (spacemacs//ensure-byte-compilation spacemacs--compiled-files)))
   ;; Check if revision has changed.
   (spacemacs//revision-check))
 
diff --git a/init.el b/init.el
index 7f2526cb6..141256730 100644
--- a/init.el
+++ b/init.el
@@ -37,18 +37,6 @@
 (load (concat spacemacs-core-directory "core-dumper.el")
       nil (not init-file-debug))
 
-;; Remove compiled core files if they become stale or Emacs version has changed.
-(load (concat spacemacs-core-directory "core-compilation.el")
-      nil (not init-file-debug))
-(load spacemacs--last-emacs-version-file t (not init-file-debug))
-(when (or (not (string= spacemacs--last-emacs-version emacs-version))
-          (spacemacs//dir-contains-stale-byte-compiled-files-p
-           spacemacs-core-directory))
-  (spacemacs//remove-byte-compiled-files-in-dir spacemacs-core-directory))
-;; Update saved Emacs version.
-(unless (string= spacemacs--last-emacs-version emacs-version)
-  (spacemacs//update-last-emacs-version))
-
 (if (not (version<= spacemacs-emacs-min-version emacs-version))
     (error (concat "Your version of Emacs (%s) is too old. "
                    "Spacemacs requires Emacs version %s or above.")
-- 
2.25.4

